diff --git a/Scripts/Services/Harvest/Mining.cs b/Scripts/Services/Harvest/Mining.cs
--- a/Scripts/Services/Harvest/Mining.cs
+++ b/Scripts/Services/Harvest/Mining.cs
@@ -8,6 +8,7 @@ namespace Server.Engines.Harvest
 {
     public class Mining : HarvestSystem
     {
+        private static double GetMiningDelay() => Config.Get("Harvest.MiningDelay", 1.0);
         private static Mining m_System;
 
         public static Mining System
@@ -62,8 +63,8 @@ namespace Server.Engines.Harvest
             oreAndStone.EffectActions = new int[] { Core.SA ? 3 : 11 };
             oreAndStone.EffectSounds = new int[] { 0x125, 0x126 };
             oreAndStone.EffectCounts = new int[] { 1 };
-            oreAndStone.EffectDelay = TimeSpan.FromSeconds(1.6);
-            oreAndStone.EffectSoundDelay = TimeSpan.FromSeconds(0.9);
+            oreAndStone.EffectDelay = TimeSpan.FromSeconds(GetMiningDelay() * 0.6);
+            oreAndStone.EffectSoundDelay = TimeSpan.FromSeconds(GetMiningDelay() * 0.4);
 
             oreAndStone.NoResourcesMessage = 503040; // There is no metal here to mine.
             oreAndStone.DoubleHarvestMessage = 503042; // Someone has gotten to the metal before you.
@@ -156,8 +157,8 @@ namespace Server.Engines.Harvest
             sand.EffectActions = new int[] { Core.SA ? 3 : 11 };
             sand.EffectSounds = new int[] { 0x125, 0x126 };
             sand.EffectCounts = new int[] { 6 };
-            sand.EffectDelay = TimeSpan.FromSeconds(1.6);
-            sand.EffectSoundDelay = TimeSpan.FromSeconds(0.9);
+            sand.EffectDelay = TimeSpan.FromSeconds(GetMiningDelay() * 0.6);
+            sand.EffectSoundDelay = TimeSpan.FromSeconds(GetMiningDelay() * 0.4);
 
             sand.NoResourcesMessage = 1044629; // There is no sand here to mine.
             sand.DoubleHarvestMessage = 1044629; // There is no sand here to mine.
