diff --git a/Scripts/Services/Harvest/Lumberjacking.cs b/Scripts/Services/Harvest/Lumberjacking.cs
--- a/Scripts/Services/Harvest/Lumberjacking.cs
+++ b/Scripts/Services/Harvest/Lumberjacking.cs
@@ -7,6 +7,7 @@ namespace Server.Engines.Harvest
 {
     public class Lumberjacking : HarvestSystem
     {
+        private static double GetLumberDelay() => Config.Get("Harvest.LumberjackingDelay", 1.0);
         private static Lumberjacking m_System;
 
         public static Lumberjacking System
@@ -67,8 +68,8 @@ namespace Server.Engines.Harvest
             lumber.EffectActions = new int[] { Core.SA ? 7 : 13 };
             lumber.EffectSounds = new int[] { 0x13E };
             lumber.EffectCounts = (Core.AOS ? new int[] { 1 } : new int[] { 1, 2, 2, 2, 3 });
-            lumber.EffectDelay = TimeSpan.FromSeconds(1.6);
-            lumber.EffectSoundDelay = TimeSpan.FromSeconds(0.9);
+            lumber.EffectDelay = TimeSpan.FromSeconds(GetLumberDelay() * 0.6);
+            lumber.EffectSoundDelay = TimeSpan.FromSeconds(GetLumberDelay() * 0.4);
 
             lumber.NoResourcesMessage = 500493; // There's not enough wood here to harvest.
             lumber.FailMessage = 500495; // You hack at the tree for a while, but fail to produce any useable wood.
