diff --git a/Scripts/Mobiles/Normal/BaseCreature.cs b/Scripts/Mobiles/Normal/BaseCreature.cs
index d7a016d..1b201b4 100644
--- a/Scripts/Mobiles/Normal/BaseCreature.cs
+++ b/Scripts/Mobiles/Normal/BaseCreature.cs
@@ -1968,12 +1968,21 @@ namespace Server.Mobiles
             }
         }
 
+		public delegate void CreatureDamageHook(BaseCreature victim, Mobile attacker, ref int totalDamage);
+		public static CreatureDamageHook OnCreatureDamageHook { get; set; }
+
         public virtual void OnBeforeDamage(Mobile from, ref int totalDamage, DamageType type)
         {
             if (type >= DamageType.Spell && RecentSetControl)
             {
                 totalDamage = 0;
             }
+
+            // Hunter's Bestiary +20% Damage Hook:
+            if (OnCreatureDamageHook != null)
+            {
+                OnCreatureDamageHook(this, from, ref totalDamage);
+            }
         }
 
         public override void OnDamage(int amount, Mobile from, bool willKill)
