diff --git a/Scripts/Items/Internal/ItemInterfaces.cs b/Scripts/Items/Internal/ItemInterfaces.cs
index cdd73f1..11396d8 100644
--- a/Scripts/Items/Internal/ItemInterfaces.cs
+++ b/Scripts/Items/Internal/ItemInterfaces.cs
@@ -79,4 +79,9 @@ namespace Server.Items
         North = 3,
         West = 4
     }
+
+    public interface IDyesTargetHandler
+    {
+        void OnDyesUsed(Mobile from, DyeTub tub);
+    }
 }
diff --git a/Scripts/Items/Tools/Dyes.cs b/Scripts/Items/Tools/Dyes.cs
index fda0ad8..3f9accb 100644
--- a/Scripts/Items/Tools/Dyes.cs
+++ b/Scripts/Items/Tools/Dyes.cs
@@ -56,7 +56,9 @@ namespace Server.Items
 
                     if (tub.Redyable)
                     {
-                        if (tub.CustomHuePicker == null)
+                        if (tub is IDyesTargetHandler handler)
+                            handler.OnDyesUsed(from, tub);
+                        else if (tub.CustomHuePicker == null)
                             from.SendHuePicker(new InternalPicker(tub));
                         else
                             from.SendGump(new CustomHuePickerGump(from, tub.CustomHuePicker, new CustomHuePickerCallback(SetTubHue), tub));
